oracle/README.md

```diff
@@ -12,11 +12,9 @@ Get metrics from Oracle Database servers in real time to visualize and monitor a
 
 #### Prerequisite
 
-To use the Oracle integration you can either use the native client (no additional install steps required) or download the Oracle JDBC driver (Linux only). To use the Oracle integration with JDBC, download the Oracle JDBC driver. If not using the JDBC method, the minimum [supported version][2] is Oracle 12c.
+To use the Oracle integration you can either use the native client (no additional install steps required), the Oracle Instant Client by using `python-oracledb` thick mode, or download the Oracle JDBC driver (Linux only). To use the Oracle integration with JDBC, download the Oracle JDBC driver. If you are not using the JDBC method, the minimum [supported version][2] is Oracle 12c.
 Due to licensing restrictions, the JDBC library is not included in the Datadog Agent, but can be downloaded directly from Oracle.
 
-With Agent v7.42.x, the Agent no longer requires or supports installing the Instant Client libraries. If you're on an older version of the Agent and want to use the Instant Client, refer to the [Oracle Instant Client][3] setup instructions.
-
 *NOTE*: Starting in v7.42.x, the Oracle integration only supports Python 3.
 
 
```

Comment:
should we provide here some instructions on how to install the instant client?

Reply:
I actually have a link mentioned in the troubleshooting section. If that doesn't suffice, I can re-add one in a more obvious location.

Reply:
We need installation instructions on the installation section, same as we have for JDBC driver

Reply:
I added the instructions from v7.41.x that we removed.